src/organizations/components/OrgProfileTab/LeaveOrg.tsx

```diff
@@ -0,0 +1,60 @@
+// Libraries
+import React, {FC, useContext} from 'react'
+import {useSelector} from 'react-redux'
+
+// Components
+import {
+  ButtonShape,
+  ComponentColor,
+  ConfirmationButton,
+  FlexBox,
+  IconFont,
+} from '@influxdata/clockface'
+
+// Selector
+import {getMe} from 'src/me/selectors'
+import {getOrg} from 'src/organizations/selectors'
+
+// Providers
+import {UsersContext} from 'src/users/context/users'
+
+// Constants
+import {CLOUD_URL} from 'src/shared/constants'
+
+// Styles
+import 'src/organizations/components/OrgProfileTab/style.scss'
+
+export const LeaveOrgButton: FC = () => {
+  const org = useSelector(getOrg)
+  const currentUserId = useSelector(getMe)?.id
+  const {handleRemoveUser} = useContext(UsersContext)
```

Comment:
Instead of using the idpe selectors here, you can use the identity selectors here, because this component only lives in CLOUD. One day it'd be nice to get rid of the idpe ones in cloud, and the fewer we have to remove at a later date, the better.

for org you can use `selectCurrentOrg`, and for the user id you can use `selectUser` (and destructure the `id` property).